.naverc

```diff
@@ -0,0 +1 @@
+10.15.0
```

Comment:
What's this file for?

Reply:
For old school [nave](https://github.com/isaacs/nave) :) – so I can run `nave auto` to switch the shell to that version of NodeJS.

Reply:
Ah ok cool. I use https://github.com/creationix/nvm

Reply:
Been meaning to switch to that for a while now… old habits die hard :D